.github/workflows/file-size-impact/report-file-size-impact.js

```diff
@@ -0,0 +1,14 @@
+const { reportFileSizeImpact, readGithubWorkflowEnv } = require("@jsenv/file-size-impact");
+
+reportFileSizeImpact({
+  ...readGithubWorkflowEnv(),
+  projectDirectoryUrl: process.env.GITHUB_WORKSPACE,
+  installCommand: "npm install",
+  buildCommand: "npm run build && rm -rf node_modules",
```

Comment:
Why `rm -rf node_modules` ?

Reply:
It's not necessary, just a local "CTRL+C" that went throught. Just fixed.